moped-api/README.md

```diff
@@ -27,15 +29,18 @@ requirements/
 
 Install the requirements in your machine, run these in order:
 
-1.First create a virutal environment in the API root folder:
+1.First create a virtual environment in the API root folder:
```

Comment:
This was the only change I intended to commit from this file, the rest of from my lint-on-save behavior that I have turned on in my code editor.

That said, I think the rest of the changes are valid markdown linting and should stay.